login.html

```diff
@@ -1,11 +1,45 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <link rel="stylesheet" href="login.css">
     <title>Welcome to the postIt! Please Log In</title>
 </head>
+
 <body>
+    <section class="wrapper">
+        <div class="content">
+
+            <header>
+                <div class="image-container">
+                    <img src="http://assets.stickpng.com/thumbs/5b06c181fad1cae04539afdd.png">
+                </div>
+                <div class="welcome-container">
+                    <h1>Welcome to postIt!</h1>
+                    <h3>Please Login!</h3>
+                </div>
+            </header>
 
+            <section>
+                <form class="login-form">
+                    <div class="input-group">
+                        <input type="email" placeholder="Email" id="email">
+                        <input type="password" placeholder="Password" id="password">
+                    </div>
+                    <div class="btn-group">
+                        <button><a id="btn-text" href="index.html">Log In</a></button>
+                        <button><a id="btn-text" href="index.html">Sign Up</a></button>
```

Comment:
```suggestion
                        <button><a id="btn-login" href="index.html">Log In</a></button>
                        <button><a id="btn-signup" href="index.html">Sign Up</a></button>
```

> The id global attribute defines an identifier (ID) which must be unique in the whole document.

 There shouldn't be duplicates here. This should be reflected in the CSS with one being in italics or something.